src/background.js

```diff
@@ -367,6 +367,7 @@ const setup = {
       log("Disabling"); 
       browser.storage.local.clear();
       await stateManager.setState("disabled");
```

Comment:
this will add back `doh-rollout.previous.trr.mode` via rememberTRRMode(). I'm not sure we need to clear the `doh-rollout.*` prefs if we're adding a few back.